metricbeat/module/couchdb/_meta/fields.yml

```diff
@@ -0,0 +1,11 @@
+- key: couchdb
+  title: "couchdb"
+  description: >
+    experimental[]
+
+    couchdb module
```

Comment:
Description and release level should be at two different variables, this should be something like:
```
description: >
  couchdb module
release: experimental
```